packages/mainsail/source/crypto/transactions/types/schemas.ts

```diff
@@ -120,6 +120,29 @@ export const delegateRegistration = extend(transactionBaseSchema, {
 	required: ["asset"],
 });
 
+export const usernameRegistration = extend(transactionBaseSchema, {
```

Comment:
this will need updating as it's currently the delegate schema. Will find you the Mainsail schema definition to compare with

Reply:
should be https://github.com/ArkEcosystem/mainsail/blob/develop/packages/crypto-transaction-username-resignation/source/versions/1.ts#L12 , which extends the generic transaction schema from https://github.com/ArkEcosystem/mainsail/blob/develop/packages/crypto-transaction/source/validation/schemas.ts

Reply:
Yeah, yeah, I know, it was too late yesterday and I wanted to have it to test the thing, that is why didn't touch the schema details, will fix that today